telemetry/csharp/AwsToolkit.Telemetry.Events.Generator.Tests/test-data/expectedCode-supplemental.txt

```diff
@@ -0,0 +1,334 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+using Amazon.AwsToolkit.Telemetry.Events;
+using Amazon.AwsToolkit.Telemetry.Events.Core;
+using System;
+using System.Collections.Generic;
+
+/// --------------------------------------------------------------------------------
+/// This file is generated from https://github.com/aws/aws-toolkit-common/tree/master/telemetry
+/// --------------------------------------------------------------------------------
+
+namespace Test
+{
+    
+    
+    /// Contains methods to record telemetry events
```

Comment:
These comments seem a little redundant

Reply:
This is the generator produced code (not changed in this PR). This should be okay since generated code can be a little cryptic when looked at, but isn't likely to be looked at often.